go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
-	github.com/cloudflare/cloudflare-go v0.66.0
+	github.com/cloudflare/cloudflare-go v0.66.1-0.20230508010110-9388783618dc
```

Comment:
please remove this from your PR. we handle these dependency changes outside of feature PRs to avoid conflicts.

Reply:
this was temporary, was waiting on the cloudflare-go release. now that it's out I'll remove the change